packages/flutter_tools/lib/src/android/android_workflow.dart

```diff
@@ -28,7 +28,8 @@ class AndroidWorkflow extends Workflow {
     androidValidator.addValidator(new Validator(
       'Android SDK',
```

Comment:
It's really this label that was confusing Jim.  He went to that page and tried to install the "Android SDK", which then presented him with more questions like "where should this be installed?" that he didn't know how to answer.  Installing "Android Studio" removes the questions from the equation.

The situation here is somewhat subtle.  We require only the SDK but the way we recommend getting the SDK is to install a larger thing that includes the SDK.